fluentui_others/src/main/java/com/microsoft/fluentui/actionbar/ActionBarLayout.kt

```diff
@@ -13,7 +13,7 @@ import kotlinx.android.synthetic.main.view_action_bar.view.*
 import java.lang.Exception
 import java.lang.IllegalStateException
 
-class ActionBarLayout @JvmOverloads constructor(context: Context, attrs: AttributeSet? = null) : TemplateView(FluentUIContextThemeWrapper(context), attrs) {
+class ActionBarLayout @JvmOverloads constructor(appContext: Context, attrs: AttributeSet? = null) : TemplateView(FluentUIContextThemeWrapper(appContext,R.style.Theme_FluentUI_Components), attrs) {
```

Comment:
cant we have default param in FluentUIContextThemeWrapper ? All the constructors seems to pass R.style.Theme_FluentUI_Components

Reply:
This is only for the others module. As the components are migrated to their own modules, R.style.Theme_FluentUI_Components will start reducing and eventually it won't be there. Hence I think its not required to make it default